pages/common/gifsicle.md

```diff
@@ -1,20 +1,32 @@
 # gifsicle
 
-> Create GIFs.
+> GIF manipulator.
 > More information: <https://www.lcdf.org/gifsicle>.
 
-- Optimise a GIF:
+- Optimize a GIF as a new file:
 
-`gifsicle --batch --optimize=3 {{amin.gif}}`
+`gifsicle {{anim.gif}} --optimize=3 -o {{optimized.gif}}`
```

Comment:
```suggestion
`gifsicle {{path/to/input.gif}} --optimize=3 -o {{path/to/optimized.gif}}`
```

Could you please apply this syntax throughout.

Reply:
Sure thing!